<!doctype html> <html lang=en-UK > <meta charset=UTF-8 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/css/main.css"> <link rel=icon  href="/assets/infra/favicon.png"> <title>Research</title> <link rel=stylesheet  href="/libs/academicons/css/academicons.min.css"> <link rel=stylesheet  href="/libs/fontello/css/fontello.css"> <link rel=stylesheet  href="/libs/line-awesome/css/line-awesome.min.css"> <header> <div class=blog-name ><a href="">Nikos Ignatiadis</a></div> <nav> <!-- NOTE: the baseurl stuff is needed only because this is a github project and therefore the baseurl is not just tlienart.github.io but in fact tlienart.github.io/JuDocExample for your own website you can safely ignore this. See also definition of the baseurl in config.md --> <ul> <li><a href="/">Home</a> <li><a href="/research">Research</a> <li><a href="/teaching">Teaching</a> <li><a href="/talks">Talks</a> </ul> <img src="/assets/infra/hamburger.svg" id=menu-icon > </nav> </header> <div class=franklin-content ><h2 id=publications_and_preprints ><a href="#publications_and_preprints" class=header-anchor >Publications and Preprints</a></h2> <h3 id=statistical_methodology_empirical_bayes ><a href="#statistical_methodology_empirical_bayes" class=header-anchor >Statistical methodology: Empirical Bayes</a></h3> <ol> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Stefan Wager</span> <span class=year >2022</span><br> <span class=title >Confidence Intervals for Nonparametric Empirical Bayes Analysis.</span><br> <span class=journal >Journal of the American Statistical Association, Theory and Methods (forthcoming).</span><br> <i> Selected as a <b>discussion paper</b> by the editors of JASA. The discussion will take place at JSM 2022. </i> <br> <span class=tooltip > <a href=https://www.tandfonline.com/doi/abs/10.1080/01621459.2021.2008403><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href=https://arxiv.org/abs/1902.02774><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/empirical-bayes-confidence-intervals-paper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/Empirikos.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Sujayam Saha</span>, <span class=person >Dennis L. Sun</span>, <span class=person >Omkar Muralidharan</span> <span class=year >2021</span><br> <span class=title >Empirical Bayes mean estimation with nonparametric errors via order statistic regression on replicated data.</span><br> <span class=journal >Journal of the American Statistical Association, Theory and Methods (forthcoming).</span><br> <span class=tooltip > <a href=https://www.tandfonline.com/doi/full/10.1080/01621459.2021.1967164><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href=https://arxiv.org/abs/1911.05970><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/AuroraPaper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/Aurora.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Panagiotis Lolas</span> <span class=year >2021</span><br> <span class=title >Ïƒ-Ridge: group-regularized ridge regression via empirical Bayes noise level cross-validation.</span><br> <span class=journal >Working paper.</span><br> <span class=tooltip > <a href=https://arxiv.org/abs/2010.15817><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/SigmaRidgeRegression.jl/tree/master/reproduction_code><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/SigmaRidgeRegression.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Stefan Wager</span> <span class=year >2019</span><br> <span class=title >Covariate-Powered Empirical Bayes Estimation.</span><br> <span class=journal >Advances in Neural Information Processing Systems 32 (NeurIPS 2019).</span><br> <span class=tooltip > <a href=https://papers.nips.cc/paper/9157-covariate-powered-empirical-bayes-estimation><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Conference paper</span> </span> <span class=tooltip > <a href=https://arxiv.org/abs/1906.01611><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://nignatiadis.github.io/assets/posters/neurips_2019_covariate_ebayes.pdf><i class="las la-chalkboard la-lg"></i></a> <span class=tooltiptext >Poster</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/EBCrossFitPaper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/EBayes.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> </ol> <h3 id=statistical_methodology_multiple_testing ><a href="#statistical_methodology_multiple_testing" class=header-anchor >Statistical methodology: Multiple testing</a></h3> <ol> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Wolfgang Huber</span> <span class=year >2021</span><br> <span class=title >Covariate powered cross-weighted multiple testing.</span><br> <span class=journal >Journal of the Royal Statistical Society: Series B (JRSS-B).</span><br> <span class=tooltip > <a href=https://doi.org/10.1111/rssb.12411><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href=https://arxiv.org/abs/1701.05179><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/Huber-group-EMBL/covariate-powered-cross-weighted-multiple-testing><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=http://bioconductor.org/packages/IHW><i class="lab la-r-project la-lg"></i></a> <span class=tooltiptext >R package</span> </span> <li> <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Bernd Klaus</span>, <span class=person >Judith Zaugg</span>, <span class=person >Wolfgang Huber</span> <span class=year >2016</span><br> <span class=title >Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.</span><br> <span class=journal >Nature methods, 13(7), 577.</span><br> <span class=tooltip > <a href=https://www.nature.com/articles/nmeth.3885><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href=http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=http://bioconductor.org/packages/IHW><i class="lab la-r-project la-lg"></i></a> <span class=tooltiptext >R package</span> </span> </ol> <h3 id=statistical_methodology_causal_inference ><a href="#statistical_methodology_causal_inference" class=header-anchor >Statistical methodology: Causal inference</a></h3> <ol> <li> <span class=person >Dean Eckles</span>, <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Stefan Wager</span>, <span class=person >Han Wu</span> <span class=year >2022</span><br> <span class=title >Noise-Induced Randomization in Regression Discontinuity Designs.</span><br> <span class=journal >Working paper.</span><br> <span class=tooltip > <a href=https://arxiv.org/abs/2004.09458><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/noise-induced-randomization-paper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/RegressionDiscontinuity.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> <li> <span class=person >Yizhe Xu</span>, <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Erik Sverdrup</span>, <span class=person >Scott Fleming</span>, <span class=person >Stefan Wager</span>, <span class=person >Nigam H. Shah</span> <span class=year >2022</span><br> <span class=title >Treatment Heterogeneity with Survival Outcomes.</span><br> <span class=journal >Chapter in: Handbook of Matching and Weighting Adjustments for Causal Inference. CRC Press (forthcoming).</span><br> <span class=tooltip > <a href=https://arxiv.org/abs/2207.07758><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/som-shahlab/survlearners><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/som-shahlab/survlearners/tree/master/r-package/survlearners><i class="lab la-r-project la-lg"></i></a> <span class=tooltiptext >R package</span> </span> </ol> <h3 id=applied_collaborations ><a href="#applied_collaborations" class=header-anchor >Applied Collaborations</a></h3> <ol> <li> <span class=person >Stephen Pfohl</span>, <span class=person >Yizhe Xu</span>, <span class=person >Agata Foryciarz</span>, <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Julian Genkins</span>, <span class=person >Nigam H. Shah</span> <span class=year >2022</span><br> <span class=title >Net benefit, calibration, threshold selection, and training objectives for algorithmic fairness in healthcare.</span><br> <span class=journal >ACM Conference on Fairness, Accountability, and Transparency (FAccT'22).</span><br> <span class=tooltip > <a href=https://dl.acm.org/doi/10.1145/3531146.3533166><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Conference paper</span> </span> <span class=tooltip > <a href=https://arxiv.org/abs/2202.01906><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href=https://github.com/som-shahlab/net_benefit_ascvd><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <li> <span class=person >Loukia Karacosta</span>, <span class=person >Benedict Anchang</span>, <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Samuel Kimmey</span>, <span class=person >Jalen Benson</span>, <span class=person >Joseph Shrager</span>, <span class=person >Robert Tibshirani</span>, <span class=person >Sean Bendall</span>, <span class=person >Sylvia K. Plevritis</span> <span class=year >2019</span><br> <span class=title >Mapping lung cancer epithelial-mesenchymal transition states and trajectories with single-cell resolution.</span><br> <span class=journal >Nature communications, 10, 5887.</span><br> <span class=tooltip > <a href=https://www.nature.com/articles/s41467-019-13441-6><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/TRACER><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href=https://github.com/nignatiadis/Empirikos.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> <li> <span class=person >Ralf Beer</span>, <span class=person >Konrad Herbst</span>, <span class=person >Nikolaos Ignatiadis</span>, <span class=person >Ilia Kats</span>, <span class=person >Lorenz Adlung</span>, <span class=person >Hannah Meyer</span>, <span class=person >Dominik Niopek</span>, <span class=person >Tania Christiansen</span>, <span class=person >Fanny Georgi</span>, <span class=person >Nils Kurzawa</span>, <span class=person >Johanna Meichsner</span>, <span class=person >Sophie Rabe</span>, <span class=person >Anja Riedel</span>, <span class=person >Joshua Sachs</span>, <span class=person >Julia Schessner</span>, <span class=person >Florian Schmidt</span>, <span class=person >Philipp Walch</span>, <span class=person >Katharina Niopek</span>, <span class=person >Tim Heinemann</span>, <span class=person >Roland Eils</span>, <span class=person >Barbara Di Ventura</span> <span class=year >2014</span><br> <span class=title >Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange.</span><br> <span class=journal >Molecular BioSystems, 10(7), 1709-1718.</span><br> <span class=tooltip > <a href=https://doi.org/10.1039/C3MB70594C><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> </ol> <div class=page-foot > <div class=copyright > Last modified: August 02, 2022. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a>. </div> </div> </div>