<!doctype html> <html lang=en-UK > <meta charset=UTF-8 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/css/main.css"> <link rel=icon  href="/assets/infra/favicon.png"> <title>Research</title> <header> <div class=blog-name ><a href="">Nikos Ignatiadis</a></div> <nav> <!-- NOTE: the baseurl stuff is needed only because this is a github project and therefore the baseurl is not just tlienart.github.io but in fact tlienart.github.io/JuDocExample for your own website you can safely ignore this. See also definition of the baseurl in config.md --> <ul> <li><a href="/">Home</a> <li><a href="/research">Research	</a> <li><a href="/teaching">Teaching</a> </ul> <img src="/assets/infra/hamburger.svg" id=menu-icon > </nav> </header> <div class=franklin-content ><h2 id=publications_and_preprints ><a href="#publications_and_preprints">Publications and Preprints</a></h2> <h3 id=statistical_methodology_empirical_bayes ><a href="#statistical_methodology_empirical_bayes">Statistical methodology: Empirical Bayes</a></h3> <ol> <li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#41;. <strong>Bias-Aware Confidence Intervals for Empirical Bayes Analysis.</strong> <br/> <a href="https://arxiv.org/abs/1902.02774">&#91;arXiv&#93;</a><a href="https://github.com/nignatiadis/MCEBPaper">&#91;reproduction code&#93;</a><a href="https://github.com/nignatiadis/MinimaxCalibratedEBayes.jl">&#91;software&#93;</a></p> <li><p>Ignatiadis, N., Saha, S., Sun D. L., &amp; Muralidharan, O. &#40;2019&#41;. <strong>Empirical Bayes mean estimation with nonparametric errors via order statistic regression.</strong> <br/><a href="https://arxiv.org/abs/1911.05970">&#91;arXiv&#93;</a><a href="https://github.com/nignatiadis/AuroraPaper">&#91;reproduction code&#93;</a></p> <li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#41;. <strong>Covariate-Powered Empirical Bayes Estimation.</strong> Advances in Neural Information Processing Systems 32 &#40;NeurIPS 2019&#41;.<br/> <a href="https://papers.nips.cc/paper/9157-covariate-powered-empirical-bayes-estimation">&#91;conference&#93;</a><a href="https://arxiv.org/abs/1906.01611">&#91;arXiv&#93;</a><a href="https://nignatiadis.github.io/assets/posters/neurips_2019_covariate_ebayes.pdf">&#91;poster&#93;</a><a href="https://github.com/nignatiadis/EBCrossFitPaper">&#91;reproduction code&#93;</a><a href="https://github.com/nignatiadis/EBayes.jl">&#91;software&#93;</a></p> </ol> <h3 id=statistical_methodology_multiple_testing ><a href="#statistical_methodology_multiple_testing">Statistical methodology: Multiple testing</a></h3> <ol> <li><p>Ignatiadis, N., &amp; Huber, W. &#40;2018&#41;. <strong>Covariate powered cross-weighted multiple testing.</strong> <br/> <a href="https://arxiv.org/abs/1701.05179">&#91;arXiv&#93;</a><a href="https://github.com/nignatiadis/IHWStatsPaper">&#91;reproduction code&#93;</a><a href="http://bioconductor.org/packages/IHW">&#91;software&#93;</a></p> <li><p>Ignatiadis, N., Klaus, B., Zaugg, J. B., &amp; Huber, W. &#40;2016&#41;. <strong>Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.</strong> Nature methods, 13&#40;7&#41;, 577. <br/> <a href="https://www.nature.com/articles/nmeth.3885">&#91;journal&#93;</a><a href="http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html">&#91;reproduction code&#93;</a><a href="http://bioconductor.org/packages/IHW">&#91;software&#93;</a></p> </ol> <h3 id=statistical_methodology_causal_inference ><a href="#statistical_methodology_causal_inference">Statistical methodology: Causal inference</a></h3> <ol> <li><p>Eckles, D., Ignatiadis, N., Wager, S., &amp; Wu, H. &#40;2020&#41;. <strong>Noise-Induced Randomization in Regression Discontinuity Designs</strong> <br/> <a href="https://arxiv.org/abs/2004.09458">&#91;arXiv&#93;</a></p> </ol> <h3 id=applied_collaborations ><a href="#applied_collaborations">Applied Collaborations</a></h3> <ol> <li><p>Karacosta, L. G., Anchang, B., Ignatiadis, N., Kimmey, S. C., Benson, J. A., Shrager, J. B., Tibshirani, R., Bendall, S.C. &amp; Plevritis, S. K. &#40;2019&#41;. <strong>Mapping lung cancer epithelial-mesenchymal transition states and trajectories with single-cell resolution.</strong> Nature communications, 10, 5887. <br/> <a href="https://www.nature.com/articles/s41467-019-13441-6">&#91;journal&#93;</a><a href="https://www.biorxiv.org/content/10.1101/570341v1">&#91;bioRxiv&#93;</a><a href="https://github.com/nignatiadis/TRACER">&#91;reproduction code&#93;</a></p> <li><p>Beer, R., Herbst, K., Ignatiadis, N., Kats, I., Adlung, L., Meyer, H., Niopek, D., Christiansen, T., Georgi, F., Kurzawa, N., Meichsner, J., Rabe, S., Riedel, A., Sachs, J., Schessner, J., Schmidt, F., Walch, P., Niopek, K., Heinemann, T., Eils, R., &amp; Di Ventura, B. &#40;2014&#41;. <strong>Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange.</strong> Molecular BioSystems, 10&#40;7&#41;, 1709-1718. <br/> <a href="https://pubs.rsc.org/--/content/articlehtml/2014/mb/c3mb70594c">&#91;journal&#93;</a></p> </ol> <h2 id=software ><a href="#software">Software</a></h2> <ol> <li><p><a href="http://bioconductor.org/packages/IHW">IHW</a>: A R/Bioconductor package implementing the Independent Hypothesis Weighting method.</p> <li><p><a href="http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html">IHWpaper</a>: A package reproducing all analyses for the Independent Hypothesis Weighting publications.</p> <li><p><a href="https://github.com/nignatiadis/SmoothingSplines.jl">SmoothingSplines.jl</a>: A statistical package for nonparametric regression via Smoothing Splines in Julia.</p> <li><p><a href="https://github.com/nignatiadis/EBayes.jl">EBayes.jl</a>: A Julia package that implements the empirical Bayes with cross-fitting method &#40;EBCF&#41;.</p> </ol> <div class=page-foot > <div class=copyright > Last modified: October 30, 2020. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a>. </div> </div> </div>