<!doctype html> <html lang=en-UK > <meta charset=UTF-8 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/css/main.css"> <link rel=icon  href="/assets/infra/favicon.png"> <title>Research</title> <link rel=stylesheet  href="/libs/academicons/css/academicons.min.css"> <link rel=stylesheet  href="/libs/fontello/css/fontello.css"> <link rel=stylesheet  href="/libs/line-awesome/css/line-awesome.min.css"> <header> <div class=blog-name ><a href="">Nikos Ignatiadis</a></div> <nav> <!-- NOTE: the baseurl stuff is needed only because this is a github project and therefore the baseurl is not just tlienart.github.io but in fact tlienart.github.io/JuDocExample for your own website you can safely ignore this. See also definition of the baseurl in config.md --> <ul> <li><a href="/">Home</a> <li><a href="/research">Research</a> <li><a href="/teaching">Teaching</a> <li><a href="/talks">Talks</a> </ul> <img src="/assets/infra/hamburger.svg" id=menu-icon > </nav> </header> <div class=franklin-content ><h2 id=publications_and_preprints ><a href="#publications_and_preprints" class=header-anchor >Publications and Preprints</a></h2> <h3 id=statistical_methodology_empirical_bayes ><a href="#statistical_methodology_empirical_bayes" class=header-anchor >Statistical methodology: Empirical Bayes</a></h3> <ol> <li><p>Ignatiadis, N., Saha, S., Sun D. L., &amp; Muralidharan, O. &#40;2021&#41;. <strong>Empirical Bayes mean estimation with nonparametric errors via order statistic regression on replicated data.</strong> Journal of the American Statistical Association &#40;forthcoming&#41;. <br/> <span class=tooltip > <a href= https://www.tandfonline.com/doi/full/10.1080/01621459.2021.1967164><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href= https://arxiv.org/abs/1911.05970><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/AuroraPaper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/Aurora.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> </p> <li><p>Ignatiadis, N., &amp; Lolas, P. &#40;2020&#43;&#41;. <strong>Ïƒ-Ridge: group-regularized ridge regression via empirical Bayes noise level cross-validation.</strong> <br/> <span class=tooltip > <a href= https://arxiv.org/abs/2010.15817><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/SigmaRidgeRegression.jl/tree/master/reproduction_code><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/SigmaRidgeRegression.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> </p> <li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#43;&#41;. <strong>Confidence Intervals for Nonparametric Empirical Bayes Analysis.</strong> <br/> <span class=tooltip > <a href= https://arxiv.org/abs/1902.02774><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/empirical-bayes-confidence-intervals-paper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/Empirikos.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> </p> </ol> <ol start=4 > <li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#41;. <strong>Covariate-Powered Empirical Bayes Estimation.</strong> Advances in Neural Information Processing Systems 32 &#40;NeurIPS 2019&#41;.<br/> <span class=tooltip > <a href= https://papers.nips.cc/paper/9157-covariate-powered-empirical-bayes-estimation><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Conference paper</span> </span> <span class=tooltip > <a href= https://arxiv.org/abs/1906.01611><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href= https://nignatiadis.github.io/assets/posters/neurips_2019_covariate_ebayes.pdf><i class="las la-chalkboard la-lg"></i></a> <span class=tooltiptext >Poster</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/EBCrossFitPaper><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/EBayes.jl><i class=icon-julia-dots ></i></a> <span class=tooltiptext >Julia package</span> </span> </p> </ol> <h3 id=statistical_methodology_multiple_testing ><a href="#statistical_methodology_multiple_testing" class=header-anchor >Statistical methodology: Multiple testing</a></h3> <ol> <li><p>Ignatiadis, N., &amp; Huber, W. &#40;2021&#41;. <strong>Covariate powered cross-weighted multiple testing.</strong> Journal of the Royal Statistical Society: Series B &#40;JRSS-B&#41;.<br/> <span class=tooltip > <a href= https://doi.org/10.1111/rssb.12411><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href= https://arxiv.org/abs/1701.05179><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> <span class=tooltip > <a href= https://github.com/Huber-group-EMBL/covariate-powered-cross-weighted-multiple-testing><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= http://bioconductor.org/packages/IHW><i class="lab la-r-project la-lg"></i></a> <span class=tooltiptext >R package</span> </span> </p> <li><p>Ignatiadis, N., Klaus, B., Zaugg, J. B., &amp; Huber, W. &#40;2016&#41;. <strong>Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.</strong> Nature methods, 13&#40;7&#41;, 577. <br/> <span class=tooltip > <a href= https://www.nature.com/articles/nmeth.3885><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href= http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> <span class=tooltip > <a href= http://bioconductor.org/packages/IHW><i class="lab la-r-project la-lg"></i></a> <span class=tooltiptext >R package</span> </span> </p> </ol> <h3 id=statistical_methodology_causal_inference ><a href="#statistical_methodology_causal_inference" class=header-anchor >Statistical methodology: Causal inference</a></h3> <ol> <li><p>Eckles, D., Ignatiadis, N., Wager, S., &amp; Wu, H. &#40;2020&#43;&#41;. <strong>Noise-Induced Randomization in Regression Discontinuity Designs</strong> <br/> <span class=tooltip > <a href= https://arxiv.org/abs/2004.09458><i class="ai ai-arxiv ai-lg"></i></a> <span class=tooltiptext >arXiv</span> </span> </p> </ol> <h3 id=applied_collaborations ><a href="#applied_collaborations" class=header-anchor >Applied Collaborations</a></h3> <ol> <li><p>Karacosta, L. G., Anchang, B., Ignatiadis, N., Kimmey, S. C., Benson, J. A., Shrager, J. B., Tibshirani, R., Bendall, S.C. &amp; Plevritis, S. K. &#40;2019&#41;. <strong>Mapping lung cancer epithelial-mesenchymal transition states and trajectories with single-cell resolution.</strong> Nature communications, 10, 5887. <br/> <span class=tooltip > <a href= https://www.nature.com/articles/s41467-019-13441-6><i class="ai ai-open-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> <span class=tooltip > <a href= https://www.biorxiv.org/content/10.1101/570341v1><i class="ai ai-biorxiv ai-lg"></i></a> <span class=tooltiptext >bioRxiv</span> </span> <span class=tooltip > <a href= https://github.com/nignatiadis/TRACER><i class="las la-code la-lg"></i></a> <span class=tooltiptext >Reproduction code</span> </span> </p> <li><p>Beer, R., Herbst, K., Ignatiadis, N., Kats, I., Adlung, L., Meyer, H., Niopek, D., Christiansen, T., Georgi, F., Kurzawa, N., Meichsner, J., Rabe, S., Riedel, A., Sachs, J., Schessner, J., Schmidt, F., Walch, P., Niopek, K., Heinemann, T., Eils, R., &amp; Di Ventura, B. &#40;2014&#41;. <strong>Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange.</strong> Molecular BioSystems, 10&#40;7&#41;, 1709-1718. <br/> <span class=tooltip > <a href= https://pubs.rsc.org/--/content/articlehtml/2014/mb/c3mb70594c><i class="ai ai-closed-access ai-lg"></i></a> <span class=tooltiptext >Journal paper</span> </span> </p> </ol> <div class=page-foot > <div class=copyright > Last modified: August 19, 2021. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a>. </div> </div> </div>