@def title = "Research"


## Publications and Preprints

### Statistical methodology: Empirical Bayes
1) Ignatiadis, N., & Wager, S. (2019). **Bias-Aware Confidence Intervals for Empirical Bayes Analysis.** \\ [[arXiv]](https://arxiv.org/abs/1902.02774)[[reproduction code]](https://github.com/nignatiadis/MCEBPaper)[[software]](https://github.com/nignatiadis/MinimaxCalibratedEBayes.jl)

2) Ignatiadis, N., Saha, S., Sun D. L., & Muralidharan, O. (2019).  **Empirical Bayes mean estimation with nonparametric errors via order statistic regression.** \\[[arXiv]](https://arxiv.org/abs/1911.05970)[[reproduction code]](https://github.com/nignatiadis/AuroraPaper)

3) Ignatiadis, N., & Wager, S. (2019). **Covariate-Powered Empirical Bayes Estimation.** Advances in Neural Information Processing Systems 32 (NeurIPS 2019).\\  [[conference]](https://papers.nips.cc/paper/9157-covariate-powered-empirical-bayes-estimation)[[arXiv]](https://arxiv.org/abs/1906.01611)[[poster]](https://nignatiadis.github.io/assets/posters/neurips_2019_covariate_ebayes.pdf)[[reproduction code]](https://github.com/nignatiadis/EBCrossFitPaper)[[software]](https://github.com/nignatiadis/EBayes.jl)

### Statistical methodology: Multiple testing

1) Ignatiadis, N., & Huber, W. (2018). **Covariate powered cross-weighted multiple testing.** \\ [[arXiv]](https://arxiv.org/abs/1701.05179)[[reproduction code]](https://github.com/nignatiadis/IHWStatsPaper)[[software]](http://bioconductor.org/packages/IHW)

2) Ignatiadis, N., Klaus, B., Zaugg, J. B., & Huber, W. (2016). **Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.** Nature methods, 13(7), 577. \\ [[journal]](https://www.nature.com/articles/nmeth.3885)[[reproduction code]](http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html)[[software]](http://bioconductor.org/packages/IHW)

### Statistical methodology: Causal inference

1) Eckles, D., Ignatiadis, N., Wager, S., & Wu, H. (2020). **Noise-Induced Randomization in Regression Discontinuity Designs** \\ [[arXiv]](https://arxiv.org/abs/2004.09458)

### Applied Collaborations

1) Karacosta, L. G., Anchang, B., Ignatiadis, N., Kimmey, S. C., Benson, J. A., Shrager, J. B., Tibshirani, R., Bendall, S.C. & Plevritis, S. K. (2019). **Mapping lung cancer epithelial-mesenchymal transition states and trajectories with single-cell resolution.** Nature communications, 10, 5887. \\ [[journal]](https://www.nature.com/articles/s41467-019-13441-6)[[bioRxiv]](https://www.biorxiv.org/content/10.1101/570341v1)[[reproduction code]](https://github.com/nignatiadis/TRACER)

2) Beer, R., Herbst, K., Ignatiadis, N., Kats, I., Adlung, L., Meyer, H., Niopek, D., Christiansen, T., Georgi, F., Kurzawa, N., Meichsner, J., Rabe, S., Riedel, A., Sachs, J., Schessner, J., Schmidt, F., Walch, P., Niopek, K.,  Heinemann, T., Eils, R., & Di Ventura, B. (2014). **Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange.** Molecular BioSystems, 10(7), 1709-1718. \\ [[journal]](https://pubs.rsc.org/--/content/articlehtml/2014/mb/c3mb70594c)

## Software

1) [IHW](http://bioconductor.org/packages/IHW): A R/Bioconductor package implementing the Independent Hypothesis Weighting method.

2) [IHWpaper](http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html): A package reproducing all analyses for the Independent Hypothesis Weighting publications.

3) [SmoothingSplines.jl](https://github.com/nignatiadis/SmoothingSplines.jl): A statistical package for nonparametric regression via Smoothing Splines in Julia.

4) [EBayes.jl](https://github.com/nignatiadis/EBayes.jl): A Julia package that implements the empirical Bayes with cross-fitting method (EBCF).
