<!doctype html>
<html lang="en-UK">
	<head>
		<meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
		<link rel="stylesheet" href="/css/main.css">
        <link rel="icon" href="/assets/infra/favicon.png">
		 <title>Research</title>	
		 <!-- Un-minified script so that can play a bit -->
<link rel="stylesheet" href="/libs/katex/katex.min.css">
 		
        
	</head>
    <body>
        <header>
            <div class="blog-name"><a href="">Nikos Ignatiadis</a></div>
            <nav>
				<!--
				NOTE: the baseurl stuff is needed only because this is a github project and therefore the baseurl is not just tlienart.github.io but in fact tlienart.github.io/JuDocExample
				for your own website you can safely ignore this.
				See also definition of the baseurl in config.md
				-->
                <ul>
                    <li><a href="/">Home</a></li>
                    <li><a href="/pub/research.html">Research	</a></li>
                    <li><a href="/pub/teaching.html">Teaching</a></li>                </ul>
                <img src="/assets/infra/hamburger.svg" id="menu-icon">
            </nav>
        </header>

<!-- Content appended here -->

<div class="jd-content">
<h2>Publications and Preprints</h2>
<h3>Statistical methodology</h3>
<ol>
<li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#41;. Covariate-Powered Empirical Bayes Estimation. <a href="https://arxiv.org/abs/1906.01611">arXiv:1906.01611.</a></p>
</li>
<li><p>Ignatiadis, N., &amp; Wager, S. &#40;2019&#41;. Bias-Aware Confidence Intervals for Empirical Bayes Analysis. <a href="https://arxiv.org/abs/1902.02774">arXiv:1902.02774.</a></p>
</li>
<li><p>Ignatiadis, N., &amp; Huber, W. &#40;2018&#41;. Covariate powered cross-weighted multiple testing. <a href="https://arxiv.org/abs/1701.05179">arXiv:1701.05179.</a></p>
</li>
<li><p>Ignatiadis, N., Klaus, B., Zaugg, J. B., &amp; Huber, W. &#40;2016&#41;. Data-driven hypothesis weighting increases detection power in genome-scale multiple testing. Nature methods, 13&#40;7&#41;, 577.</p>
</li>
</ol>
<h3>Applied Collaborations</h3>
<ol>
<li><p>Karacosta, L. G., Anchang, B., Ignatiadis, N., Kimmey, S. C., Benson, J. A., Shrager, J. B., Tibshirani, R., Bendall, S.C. &amp; Plevritis, S. K. &#40;2019&#41;. Mapping Lung Cancer Epithelial-Mesenchymal Transition States and Trajectories with Single-Cell Resolution. <a href="https://www.biorxiv.org/content/10.1101/570341v1">bioRxiv, 570341.</a></p>
</li>
<li><p>Beer, R., Herbst, K., Ignatiadis, N., Kats, I., Adlung, L., Meyer, H., Niopek, D., Christiansen, T., Georgi, F., Kurzawa, N., Meichsner, J., Rabe, S., Riedel, A., Sachs, J., Schessner, J., Schmidt, F., Walch, P., Niopek, K.,  Heinemann, T., Eils, R., &amp; Di Ventura, B. &#40;2014&#41;. Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange. Molecular BioSystems, 10&#40;7&#41;, 1709-1718.</p>
</li>
</ol>
<h2>Software</h2>
<ol>
<li><p><a href="http://bioconductor.org/packages/IHW">IHW</a>: A R/Bioconductor package implementing the Independent Hypothesis Weighting method.</p>
</li>
<li><p><a href="http://bioconductor.org/packages/devel/data/experiment/html/IHWpaper.html">IHWpaper</a>: A package reproducing all analyses for the Independent Hypothesis Weighting publications.</p>
</li>
<li><p><a href="https://github.com/nignatiadis/SmoothingSplines.jl">SmoothingSplines.jl</a>: A statistical package for nonparametric regression via Smoothing Splines in Julia.</p>
</li>
</ol>

<div class="page-foot">
		<div class="copyright">
				Last modified: June 05, 2019. Website built with <a href="https://github.com/tlienart/JuDoc.jl">JuDoc.jl</a>.
		</div>
</div>
</div>
<!-- CONTENT ENDS HERE -->
        
                



        
    </body>
</html>